scalableminds/webknossos
# Added info text to upload zone when upload is disabled

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- open a tracing view
- if the user has data manipulating permissions (update permissions) the drag'n'drop should work as before
- if the user does not have the needed permissions (like in view mode) the upload zone should show a error message and dropping will only close the drop zone and not trying to parse the file or something like that

### Issues:
- fixes [#2977](https://github.com/scalableminds/webknossos/issues/2977)

------
